pkg/cloud/cloud.go

```diff
@@ -208,6 +236,51 @@ func (c *cloud) DescribeFileSystem(ctx context.Context, fileSystemId string) (fs
 	}, nil
 }
 
+func (c *cloud) DescribeMountTargets(ctx context.Context, fileSystemId, azName string) (fs *MountTarget, err error) {
+	describeMtInput := &efs.DescribeMountTargetsInput{FileSystemId: &fileSystemId}
+	klog.V(5).Infof("Calling DescribeMountTargets with input: %+v", *describeMtInput)
```

Comment:
This is info level log? Should it be a debug level?

Reply:
There isn't a specific "debug level". You can increase/decrease verbosity of your logs using klogs's log level.